README.rst

```diff
@@ -70,9 +70,74 @@ We appreciate any help!
 BUILD & DEVELOPMENT
 -------------------
 
-Run ``python3 setup.py install``
+1. Clone the *coala-html* repository
 
-Note: You need to have bower installed before you may install the python packages.
+::
+
+  $ git clone https://github.com/coala/coala-html.git
+
+2. Install latest version of **Node.js**
+
+- *Configure Node.js PPA*
+
+::
+
+    $ sudo apt-get install curl python-software-properties
+    $ curl -sL https://deb.nodesource.com/setup_11.x | sudo bash -
+
+- *Install Node.js and npm*
+
+::
+
+  $ sudo apt-get install nodejs
+
+- *Check Node.js and npm version*
+
+  You can find more details about current version on node.js `official website
+  <https://nodejs.org/>`__.
+
+::
+
+    $ node -v
+    v11.6.0
+
+    $ npm -v
+    6.5.0
+
+3. Install **bower** to build static web pages
+
+::
+
+    $ sudo npm install -g bower
+
+4. Change the directory to *coala-html* and run following command to install dependencies
+
+::
+
+    $ python setup.py install
+
+5. Install **coala-bears**
+
+::
+
+    $ pip install coala-bears
+
+6. Allow permissions to **root user** to install node packages
```

Comment:
Running `coala-html --dir ..\tests` automatically installs the packages in `~` directory for which it need root access to install packages. Otherwise, if not provided, it will throw an error and cancel the installation.

Reply:
Installing node modules with the ```-g``` flag installs them globally under ```/usr/lib``` and if the flag is not specified it is installed in the current folder. You need to find the exact command that installs the packages and maybe fix that, because, giving root permissions inside ```/home/``` is **never** advised.